pkg/config/config.go

```diff
@@ -478,6 +479,11 @@ type RuntimeConfig struct {
 	// Option to disable hostport mapping in CRI-O
 	// Default value is 'false'
 	DisableHostPortMapping bool `toml:"disable_hostport_mapping"`
+
+	// Option to specify the timezone inside the container.
+	// If set to "Local," it means the container will use
+	// the same timezone as the host machine.
```

Comment:
The overall behavior should be also documented in the CLI and markdown docs. I think we can also state the unsetting the field refers to UTC. I'd say we use `local` instead of `Local` and/or lowercase the input.